src/libsyncengine/jobs/network/API_v2/downloadjob.cpp

```diff
@@ -514,74 +514,57 @@ bool DownloadJob::moveTmpFile(const SyncPath &tmpPath, bool &restartSync) {
         retry = false;
 #endif
 
-        IoError ioError = IoError::Success;
-        IoHelper::moveItem(tmpPath, _localpath, ioError);
-        const bool crossDeviceLink = ioError == IoError::CrossDeviceLink;
-        if (ioError != IoError::Success && !crossDeviceLink) {
-            LOGW_WARN(_logger, L"Failed to move: " << Utility::formatIoError(tmpPath, ioError) << L" to "
-                                                   << Utility::formatSyncPath(_localpath));
-            return false;
-        }
+        // Copy file content (i.e. doesn't change node id)
+        ec.clear();
```

Comment:
We can now declare `ec` here; not need to clear it anymore.

Reply:
It's in a loop (#ifdef _WIN32) but yes, anyway, the compiler will optimize that.
Done